plotly/tools.py

```diff
@@ -269,12 +269,15 @@ def get_embed(file_owner_or_url, file_id=None, width="100%", height=525):
     Keyword arguments:
     file_id (default=None) -- an int or string that can be converted to int
                               if you're using a url, don't fill this in!
+    share_key (default=None) -- a string that enables private plots to be
+                                shared. Note, you must have admin privileges
+                                on the file for this to work
```

Comment:
yeah, and "admin privileges" is misleading, if you have the URL of a graph you don't need anything. You need "admin" privledges to "enable private sharing" but not necessarily to embed.

also, `file_owner_or_url` should be able to handle the URLs copied and pasted directly from the firewall:
- https://plot.ly/~christopherp/749/col2/?share_key=wRNj7Du1fvbVtTPQcmd057
- https://plot.ly/~christopherp/749/col2?share_key=wRNj7Du1fvbVtTPQcmd057
- https://plot.ly/~christopherp/749?share_key=wRNj7Du1fvbVtTPQcmd057
- https://plot.ly/~christopherp/749/?share_key=wRNj7Du1fvbVtTPQcmd057

- https://enterprise.plot.ly/~christopherp/749/col2/?share_key=wRNj7Du1fvbVtTPQcmd057
- https://enterprise.plot.ly/~christopherp/749/col2?share_key=wRNj7Du1fvbVtTPQcmd057
- https://enterprise.plot.ly/~christopherp/749?share_key=wRNj7Du1fvbVtTPQcmd057
- https://enterprise.plot.ly/~christopherp/749/?share_key=wRNj7Du1fvbVtTPQcmd057